src/averageAges.js

```diff
@@ -37,7 +44,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womanFilteredArray = people
+    .filter(person => withChildren
+      ? people.find(child => child.mother === person.name)
```

Comment:
```suggestion
      ? people.some(child => child.mother === person.name)
```

Reply:
Good idea not to check if `person.sex === 'f'` as only a woman can be a mother 👍

Reply:
> 

Why some better then find?

Reply:
`some` returns boolean. Compare:
```js
const list = ['a', 'b', '', 'c'];
const findEmpty = list.find(el => el === '');
const hasEmpty = list.some(el => el === '');

if (findEmpty) <---- FALSE
if (hasEmpty) <---- TRUE
```